pkg/apis/kops/validation/validation.go

```diff
@@ -85,17 +85,35 @@ func validateClusterSpec(spec *kops.ClusterSpec, c *kops.Cluster, fieldPath *fie
 
 	// SSHAccess
 	for i, cidr := range spec.SSHAccess {
-		allErrs = append(allErrs, validateCIDR(cidr, fieldPath.Child("sshAccess").Index(i))...)
+		if strings.HasPrefix(cidr, "pl-") {
+			if kops.CloudProviderID(spec.CloudProvider) != kops.CloudProviderAWS {
```

Comment:
Actually your suggestion exhibits different behaviour... If we'd take your suggestion, it will need another `else if` because the `pl-` prefix is allowed when it is on AWS. So that's why I made a nested if.

Reply:
Somehow was expecting the `strings.HasPrefix(cidr, "pl-")` check to also be part of `validateCIDR()`.

Reply:
No, I actually didn't add that, as that method is used to check subnet.CIDR also where I don't think a prefix list would be allowed.

Reply:
👍